File: PRACTICA/ELIMINAR DUPLICADOS/ELIMINAR.cs
using System;
using System.Collections.Generic;

class EliminarDuplicados {
    static void Main() {
        int[] arr = {1,2,2,3,4,4,5};
        HashSet<int> set = new HashSet<int>(arr);
        Console.WriteLine("Sin duplicados: " + string.Join(", ", set));
    }
}

File: PRACTICA/ELIMINAR DUPLICADOS/ELIMINAR.cpp
#include <iostream>
#include <vector>
#include <set>
using namespace std;
int main() {
    vector<int> arr = {1,2,2,3,4,4,5};
    set<int> s(arr.begin(), arr.end());
    cout << "Sin duplicados: ";
    for (int x : s) cout << x << " ";
}
